composer.json

```diff
@@ -16,19 +16,20 @@
         }
     ],
     "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
         "php": ">=5.3.9",
-        "symfony/framework-bundle": "~2.3.27 || ~2.6.6 || ~2.7 || ~3.0",
+        "symfony/framework-bundle": "^2.3.27 || ~3.0",
```

Comment:
because the fix was never backported to those versions. i think the 3 variants is easier to read than a separate conflicts with 2.4 || 2.5 || 2.6, <2.6.6